src/main_app/public/auth/__init__.py

```
@@ -0,0 +1,9 @@
+""" """
+
+from .routes import bp_auth
+from .utils import oauth_required
+
+__all__ = [
+    "oauth_required",
+    "bp_auth",
+]
```

Ruff (0.15.18)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)